tests/vector/test_spaces.py

```diff
@@ -129,3 +133,78 @@ def test_iterate_custom_space(space, batch_space):
     for i, item in enumerate(iterator):
         assert item in space
     assert i == 3
+
+
+seeded_spaces = [
+    CustomSpace(seed=123),
+    Box(0, 10, (), seed=123),
+    Tuple([Box(0, 5, (), seed=123), Box(0, 3, (), seed=123)], seed=123),
+    Dict(
+        {"space-1": Box(0, 5, (), seed=123), "space-2": Box(0, 10, (), seed=123)},
+        seed=123,
+    ),
+    Discrete(5, seed=123),
+    MultiDiscrete([5, 3], seed=123),
+]
+
+
+@pytest.mark.parametrize(
+    "space", seeded_spaces, ids=[space.__class__.__name__ for space in seeded_spaces]
+)
+def test_batch_space_seed(space):
+    batched_space = batch_space(space)  # n=1
```

Comment:
Nit:
```suggestion
    batched_space = batch_space(space, n=1)
```

Reply:
Either this, or remove the dead-code-comment (whichever is applicable)